src/calendar.cpp

```diff
@@ -17,12 +17,12 @@ const time_duration calendar::INDEFINITELY_LONG_DURATION(
 bool calendar::is_eternal_season = false;
 int calendar::cur_season_length = 1;
 
-calendar calendar::start;
+calendar calendar::start_of_cataclysm;
 calendar calendar::turn;
 season_type calendar::initial_season;
 
 const time_point calendar::before_time_starts = time_point::from_turn( -1 );
-const time_point calendar::time_of_cataclysm = time_point::from_turn( 0 );
+const time_point calendar::turn_zero = time_point::from_turn( 0 );
```

Comment:
This isn't supposed to be have anything to do with zero. This is just an arbitrary date when the cataclysm happens. It could be implemented as turn 100 or turn -100 or anything else. Nobody should *need* to known what turn it is. One should use it for calculation of time_duration only.